src/letter_template.txt

```diff
@@ -0,0 +1,12 @@
+# You can edit this template however you like. 
+# Fields that you can use:
+# {name}
+# {amount}
+#
+#
+Dear {name},
+Thank you so much for your kind donation of {amount}.
+We here at the Ministry for Silly Walks greatly appreciate it.
+Your money will go towards creating newer sillier walks.
```

Comment:
Oh, thank heavens.  The ministry has been needing a new funding source for years now.